include/kafka/BrokerMetadata.h

```diff
@@ -111,23 +122,44 @@ struct BrokerMetadata {
     std::map<Partition, PartitionInfo>        _partitions;
 };
 
+inline std::vector<BrokerMetadata::Node>
```

Comment:
What is the benefit of std::vector<Node> than std::vector<shared_ptr<Node>>?

Reply:
I'm fine with std::vector<Node>, given it's just two ints and one small string for hostname.

Reply:
Oh, `std::vector<shared_ptr<Node>>` sounds great!

Reply:
Now commit submitted! 👌
> Oh, `std::vector<shared_ptr<Node>>` sounds great!